hack/run-locally.sh

```diff
@@ -33,11 +33,18 @@ function override_install_manifests() {
 
 # Extract environment variables for the CNO DaemonSet
 function setup_operator_env() {
+    local image_env_key=${1}
+    local plugin_image=${2}
+
     if [[ manifests/0000_70_cluster-network-operator_03_daemonset.yaml -nt "${CLUSTER_DIR}/env.sh" ]]; then
         echo "Copying environment variables from manifest to ${CLUSTER_DIR}/env.sh"
         oc --kubeconfig=hack/null-kubeconfig patch --local=true -f manifests/0000_70_cluster-network-operator_03_daemonset.yaml -p '{}' -ojsonpath='{range .spec.template.spec.containers[0].env[?(@.value)]}{.name}{"="}{.value}{"\n"}' > "${CLUSTER_DIR}/env.sh"
     fi
     sed -i -e "s/^RELEASE_VERSION=.*/RELEASE_VERSION=${RELEASE_VERSION}/" "${CLUSTER_DIR}/env.sh"
+
+    if [[ -n "${image_env_key}" ]]; then
+        sed -i -e "s#^${image_env_key}=.*#${image_env_key}=${plugin_image}#" "${CLUSTER_DIR}/env.sh"
```

Comment:
This should be `if [[ -n "${plugin_image}" ]];` I think; `${IMAGE_ENV_KEY}` always gets set